en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4214 @@
+granularity value is '1d', timeSpan will represents start of the day calculated adhering to the timezone provided in
+the TimeFilter.
+Ex: timeSpan: '2021-06-21T00:00:00.0000000+05:30'
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>auth</strong></td>
+<td valign="top"><a href="#int">Int</a></td>
+<td>
+
+Authorization related error count.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>targetConnectivity</strong></td>
+<td valign="top"><a href="#int">Int</a></td>
+<td>
+
+Target Connectivity related error count.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>throttled</strong></td>
+<td valign="top"><a href="#int">Int</a></td>
+<td>
+
+Throttling related error count.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>other</strong></td>
+<td valign="top"><a href="#int">Int</a></td>
+<td>
+
+Other errors count.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### ErrorCountByCode
+
+Represents error count for a single error status code.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>statusCode</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Error status code.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>count</strong></td>
+<td valign="top"><a href="#int">Int</a>!</td>
+<td>
+
+Error count.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### ErrorCountByCodeForAPI
+
+Represents error count for each status code for a single API.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>apiId</strong></td>
+<td valign="top"><a href="#id">ID</a>!</td>
+<td>
+
+Api Id.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>errorCountByCode</strong></td>
+<td valign="top">[<a href="#errorcountbycode">ErrorCountByCode</a>]</td>
+<td>
+
+List containing error count for each error status code. Returns an empty array if no data available.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### ErrorDetails
+
+Represents details of an error for a single timestamp. Timestamp is calculated according to the provided granularity
+value.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>apiId</strong></td>
+<td valign="top"><a href="#id">ID</a></td>
+<td>
+
+API Id.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>timeSpan</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Represents a timestamp value calculated according to the used granularity value. For an example if the used
```

Comment:
When you say "used granularity value" it is not clear as to where it is used and by what. Please check whether the following suggestion gives the same meaning.

```suggestion
Represents a timestamp value that is calculated based on the specified granularity value. For example, if the granularity value is `1d`, the timestamp represents the start of the day calculated adhering to the timezone provided in the time filter.
e.g., timeSpan: '2021-06-21T00:00:00.0000000+05:30'
```